android/src/main/java/expo/modules/xmtpreactnativesdk/XMTPModule.kt

```diff
@@ -214,6 +214,17 @@ class XMTPModule : Module() {
             client.canMessage(peerAddress)
         }
 
+        AsyncFunction("staticCanMessage") { peerAddress: String, environment: String, appVersion: String? ->
+            try {
+                logV("staticCanMessage")
+                val options = ClientOptions(api = apiEnvironments(environment, appVersion))
+                Client.canMessage(peerAddress = peerAddress, options = options)
+                true
+            } catch (e: Exception) {
+                throw XMTPException("Failed to create client: $e")
```

Comment:
This will just print the name of the class "Exception.class" I think you want the actual error message which would be like this.

```suggestion
                throw XMTPException("Failed to create client: ${e.message}")
```